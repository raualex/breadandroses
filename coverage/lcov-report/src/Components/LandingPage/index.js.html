<!doctype html>
<html lang="en">
<head>
    <title>Code coverage report for src/Components/LandingPage/index.js</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../../../prettify.css" />
    <link rel="stylesheet" href="../../../base.css" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../../../sort-arrow-sprite.png);
        }
    </style>
</head>
<body>
<div class='wrapper'>
  <div class='pad1'>
    <h1>
      <a href="../../../index.html">All files</a> / <a href="index.html">src/Components/LandingPage</a> index.js
    </h1>
    <div class='clearfix'>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Statements</span>
        <span class='fraction'>2/2</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Branches</span>
        <span class='fraction'>0/0</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Functions</span>
        <span class='fraction'>1/1</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Lines</span>
        <span class='fraction'>2/2</span>
      </div>
    </div>
  </div>
  <div class='status-line high'></div>
<pre><table class="coverage">
<tr><td class="line-count quiet">1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
32
33
34
35
36
37</td><td class="line-coverage quiet"><span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">2x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span></td><td class="text"><pre class="prettyprint lang-js">import React from 'react';
import './LandingPage.css';
import star from '../../Utils/Assets/Images/back_star.png';
import fist from '../../Utils/Assets/Images/power.svg';
import { NavLink } from 'react-router-dom'
&nbsp;
export const LandingPage = () =&gt; {
&nbsp;
  return (
    &lt;div className="sun-container"&gt;
      &lt;div className="title-container"&gt;
        &lt;h1 className="title-text"&gt;BREAD&lt;/h1&gt;
        &lt;h1 className="title-text"&gt;&amp;&lt;/h1&gt;
        &lt;h1 className="title-text"&gt;ROSES&lt;/h1&gt;
      &lt;/div&gt;
      &lt;div className="outer"&gt;
        &lt;b&gt;&lt;/b&gt;&lt;b&gt;&lt;/b&gt;&lt;b&gt;&lt;/b&gt;&lt;b&gt;&lt;/b&gt;&lt;b&gt;&lt;/b&gt;&lt;b&gt;&lt;/b&gt;&lt;b&gt;&lt;/b&gt;&lt;b&gt;&lt;/b&gt;&lt;b&gt;&lt;/b&gt;&lt;b&gt;&lt;/b&gt;
      &lt;/div&gt;
      &lt;div className="star-container"&gt;
        &lt;img src={star} className="star" alt="star" /&gt;
        &lt;img src={fist} className="fist" alt="fist" /&gt;
      &lt;/div&gt;
      &lt;div className="back-foreground"&gt;
      &lt;/div&gt;
      &lt;div className="back-cover-layer"&gt;
        &lt;h2 className="subtitle"&gt;What is Congress Doing for You?&lt;/h2&gt;
        &lt;NavLink to='/welcome'&gt;
          &lt;button className="enter-btn"&gt;
            ENTER
          &lt;/button&gt;
        &lt;/NavLink&gt;
      &lt;/div&gt;
    &lt;/div&gt;
  )
}
&nbsp;
export default LandingPage;</pre></td></tr>
</table></pre>
<div class='push'></div><!-- for sticky footer -->
</div><!-- /wrapper -->
<div class='footer quiet pad2 space-top1 center small'>
  Code coverage
  generated by <a href="https://istanbul.js.org/" target="_blank">istanbul</a> at Mon Nov 05 2018 10:31:10 GMT-0700 (MST)
</div>
</div>
<script src="../../../prettify.js"></script>
<script>
window.onload = function () {
        if (typeof prettyPrint === 'function') {
            prettyPrint();
        }
};
</script>
<script src="../../../sorter.js"></script>
</body>
</html>
