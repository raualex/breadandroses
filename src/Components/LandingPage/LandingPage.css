.back-cover-layer {
  align-items: center;
  background-color: #50241E;
  bottom: 0;
  display: flex;
  flex-direction: column;
  height: 600px;
  justify-content: center;
  position: absolute;
  width: 100%;
}

.back-foreground {
  background-color: #50241E;
  border-top: solid 8px #000;
  border-radius: 200%;
  bottom: 0;
  height: 740px;
  left: -340px;
  position: absolute;
  width: 150%;
}

.enter-btn {
  background-color: #f4c076;
  border: solid 4px #000;
  font-family: 'Black Han Sans', sans-serif;
  font-size: 2em;
  font-weight: 700;
  height: 60px;
  margin-top: 20px;
  text-align: center;
  width: 250px;
}

.enter-btn:active {
  background-color: #CC2B1D;
  color: #fff;
}

.enter-btn:hover {
  cursor: pointer;
}

.enter-btn:focus {
  outline: none;
}

.fist {
  height: 200px;
  left: 200px;
  position: absolute;
  top: 240px;
  width: 200px;
  z-index: 4;
}

.outer {
  position: relative;
  top: -150px;
  left: 210px;
  width: 1000px;
  height: 455px;
  border-radius: 500px;
  padding-top: 545px;
  z-index: 0;
  -webkit-animation-name: rotate1;
  -webkit-animation-duration:2s;
  -webkit-animation-iteration-count:infinite;
  -webkit-animation-timing-function:linear;
  -moz-animation-name: rotate1;
  -moz-animation-duration:2s;
  -moz-animation-iteration-count:infinite;
  -moz-animation-timing-function:linear;
}

.outer b {
  z-index: 0;
  display: block;
  width: 0;
  height: 0;
  border-width: 45px 500px;
  overflow: hidden;
  margin: -90px 0 0 0;
  border-color: transparent #fbe75d;
  border-style: solid;
}

.outer b:nth-child(1) { -webkit-transform:rotate(20deg);  -moz-transform:rotate(20deg);  }
.outer b:nth-child(2) { -webkit-transform:rotate(40deg);  -moz-transform:rotate(40deg);  }
.outer b:nth-child(3) { -webkit-transform:rotate(60deg);  -moz-transform:rotate(60deg);  }
.outer b:nth-child(4) { -webkit-transform:rotate(80deg);  -moz-transform:rotate(80deg);  }
.outer b:nth-child(5) { -webkit-transform:rotate(100deg); -moz-transform:rotate(100deg); }
.outer b:nth-child(6) { -webkit-transform:rotate(120deg); -moz-transform:rotate(120deg); }
.outer b:nth-child(7) { -webkit-transform:rotate(140deg); -moz-transform:rotate(140deg); }
.outer b:nth-child(8) { -webkit-transform:rotate(160deg); -moz-transform:rotate(160deg); }
.outer b:nth-child(9) { -webkit-transform:rotate(180deg); -moz-transform:rotate(180deg); }

.star {
  position: absolute;
  z-index: 3;
}

.star-container {
  left: 28%;
  position: absolute;
}

.subtitle {
  -webkit-text-stroke: 2px #000;
  color: #f4c076;
  font-size: 2.5em;
  font-weight: 700;
  font-family: 'Black Han Sans', sans-serif;
  margin-right: 15px;
  margin-top: 10px;
}

.sun-container {
  background-color: #CC2B1D;
  flex-direction: column;
  display: flex;
  position: relative;
}

.title-container {
  display: flex;
  justify-content: space-around;
  padding-left: 60px;
  position: absolute;
  width: 90%;
  z-index: 4;
}

.title-text {
  -webkit-text-stroke: 2px #000;
  color: #f4c076;
  font-family: 'Black Han Sans', sans-serif;
  font-size: 4em;
}

@-webkit-keyframes rotate1 {
    from { -webkit-transform: rotate(0deg); }
    to   { -webkit-transform: rotate(20deg); }
}
@-moz-keyframes rotate1 {
    from { -moz-transform: rotate(0deg); }
    to   { -moz-transform: rotate(20deg); }
}
