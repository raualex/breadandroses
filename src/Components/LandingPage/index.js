import React from 'react';
import './LandingPage.css';
import star from '../../Utils/Assets/Images/back_star.png';
import fist from '../../Utils/Assets/Images/power.svg';

export const LandingPage = () => {

  return (
    <div className="sun-container">
      <div className="title-container">
        <h1 className="title-text">BREAD</h1>
        <h1 className="title-text">&</h1>
        <h1 className="title-text">ROSES</h1>
      </div>
      <div className="outer">
        <b></b><b></b><b></b><b></b><b></b><b></b><b></b><b></b><b></b><b></b>
      </div>
      <div className="star-container">
        <img src={star} className="star" alt="star" />
        <img src={fist} className="fist" alt="fist" />
      </div>
      <div className="back-foreground">
      </div>
      <div className="back-cover-layer">
        <h2 className="subtitle">What is Congress Doing for You?</h2>
        <button className="enter-btn">ENTER</button>
      </div>
    </div>
  )
}

export default LandingPage;