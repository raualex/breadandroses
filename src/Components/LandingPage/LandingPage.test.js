import React from 'react';
import LandingPage from './';
import { shallow } from 'enzyme';

describe('LandingPage', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<LandingPage />)
  })
  
  it('matches the snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  });
})