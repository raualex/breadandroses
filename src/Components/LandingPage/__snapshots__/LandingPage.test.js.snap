// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingPage matches the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <LandingPage />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="title-container"
        >
          <h1
            className="title-text"
          >
            BREAD
          </h1>
          <h1
            className="title-text"
          >
            &
          </h1>
          <h1
            className="title-text"
          >
            ROSES
          </h1>
        </div>,
        <div
          className="outer"
        >
          <b />
          <b />
          <b />
          <b />
          <b />
          <b />
          <b />
          <b />
          <b />
          <b />
        </div>,
        <div
          className="star-container"
        >
          <img
            alt="star"
            className="star"
            src="back_star.png"
          />
          <img
            alt="fist"
            className="fist"
            src="power.svg"
          />
        </div>,
        <div
          className="back-foreground"
        />,
        <div
          className="back-cover-layer"
        >
          <h2
            className="subtitle"
          >
            What is Congress Doing for You?
          </h2>
          <NavLink
            activeClassName="active"
            aria-current="page"
            to="/welcome"
          >
            <button
              className="enter-btn"
            >
              ENTER
            </button>
          </NavLink>
        </div>,
      ],
      "className": "sun-container",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h1
              className="title-text"
            >
              BREAD
            </h1>,
            <h1
              className="title-text"
            >
              &
            </h1>,
            <h1
              className="title-text"
            >
              ROSES
            </h1>,
          ],
          "className": "title-container",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "BREAD",
              "className": "title-text",
            },
            "ref": null,
            "rendered": "BREAD",
            "type": "h1",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "&",
              "className": "title-text",
            },
            "ref": null,
            "rendered": "&",
            "type": "h1",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "ROSES",
              "className": "title-text",
            },
            "ref": null,
            "rendered": "ROSES",
            "type": "h1",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <b />,
            <b />,
            <b />,
            <b />,
            <b />,
            <b />,
            <b />,
            <b />,
            <b />,
            <b />,
          ],
          "className": "outer",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": "b",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": "b",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": "b",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": "b",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": "b",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": "b",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": "b",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": "b",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": "b",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": "b",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <img
              alt="star"
              className="star"
              src="back_star.png"
            />,
            <img
              alt="fist"
              className="fist"
              src="power.svg"
            />,
          ],
          "className": "star-container",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "star",
              "className": "star",
              "src": "back_star.png",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "fist",
              "className": "fist",
              "src": "power.svg",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "back-foreground",
        },
        "ref": null,
        "rendered": null,
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h2
              className="subtitle"
            >
              What is Congress Doing for You?
            </h2>,
            <NavLink
              activeClassName="active"
              aria-current="page"
              to="/welcome"
            >
              <button
                className="enter-btn"
              >
                ENTER
              </button>
            </NavLink>,
          ],
          "className": "back-cover-layer",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "What is Congress Doing for You?",
              "className": "subtitle",
            },
            "ref": null,
            "rendered": "What is Congress Doing for You?",
            "type": "h2",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "activeClassName": "active",
              "aria-current": "page",
              "children": <button
                className="enter-btn"
              >
                ENTER
              </button>,
              "to": "/welcome",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "ENTER",
                "className": "enter-btn",
              },
              "ref": null,
              "rendered": "ENTER",
              "type": "button",
            },
            "type": [Function],
          },
        ],
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="title-container"
          >
            <h1
              className="title-text"
            >
              BREAD
            </h1>
            <h1
              className="title-text"
            >
              &
            </h1>
            <h1
              className="title-text"
            >
              ROSES
            </h1>
          </div>,
          <div
            className="outer"
          >
            <b />
            <b />
            <b />
            <b />
            <b />
            <b />
            <b />
            <b />
            <b />
            <b />
          </div>,
          <div
            className="star-container"
          >
            <img
              alt="star"
              className="star"
              src="back_star.png"
            />
            <img
              alt="fist"
              className="fist"
              src="power.svg"
            />
          </div>,
          <div
            className="back-foreground"
          />,
          <div
            className="back-cover-layer"
          >
            <h2
              className="subtitle"
            >
              What is Congress Doing for You?
            </h2>
            <NavLink
              activeClassName="active"
              aria-current="page"
              to="/welcome"
            >
              <button
                className="enter-btn"
              >
                ENTER
              </button>
            </NavLink>
          </div>,
        ],
        "className": "sun-container",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h1
                className="title-text"
              >
                BREAD
              </h1>,
              <h1
                className="title-text"
              >
                &
              </h1>,
              <h1
                className="title-text"
              >
                ROSES
              </h1>,
            ],
            "className": "title-container",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "BREAD",
                "className": "title-text",
              },
              "ref": null,
              "rendered": "BREAD",
              "type": "h1",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "&",
                "className": "title-text",
              },
              "ref": null,
              "rendered": "&",
              "type": "h1",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "ROSES",
                "className": "title-text",
              },
              "ref": null,
              "rendered": "ROSES",
              "type": "h1",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <b />,
              <b />,
              <b />,
              <b />,
              <b />,
              <b />,
              <b />,
              <b />,
              <b />,
              <b />,
            ],
            "className": "outer",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "b",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "b",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "b",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "b",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "b",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "b",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "b",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "b",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "b",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": "b",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <img
                alt="star"
                className="star"
                src="back_star.png"
              />,
              <img
                alt="fist"
                className="fist"
                src="power.svg"
              />,
            ],
            "className": "star-container",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "alt": "star",
                "className": "star",
                "src": "back_star.png",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "alt": "fist",
                "className": "fist",
                "src": "power.svg",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "className": "back-foreground",
          },
          "ref": null,
          "rendered": null,
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h2
                className="subtitle"
              >
                What is Congress Doing for You?
              </h2>,
              <NavLink
                activeClassName="active"
                aria-current="page"
                to="/welcome"
              >
                <button
                  className="enter-btn"
                >
                  ENTER
                </button>
              </NavLink>,
            ],
            "className": "back-cover-layer",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "What is Congress Doing for You?",
                "className": "subtitle",
              },
              "ref": null,
              "rendered": "What is Congress Doing for You?",
              "type": "h2",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "activeClassName": "active",
                "aria-current": "page",
                "children": <button
                  className="enter-btn"
                >
                  ENTER
                </button>,
                "to": "/welcome",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "ENTER",
                  "className": "enter-btn",
                },
                "ref": null,
                "rendered": "ENTER",
                "type": "button",
              },
              "type": [Function],
            },
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
